### DISCUSSION TITLE
Copy button

### DISCUSSION THREAD

**Discussion by MustCodeAl** [massCodeIO/massCode#430] [Q&A]
A copy button would be nice, as the copy to snippet from menu is a little bit inconvenient 
a nice setting could be made allowing users to disable or enable in it the menu settings

**Comment by antonreshetov** [comment]
https://github.com/massCodeIO/massCode/discussions/380

**Reply by MustCodeAl** [reply]
Yes, I saw the thread, but it's still would be more convenient to have a visible button for some users with touch screens
